docs/core/spatial-rules.md

```diff
@@ -8,11 +8,11 @@ different [speeds](https://wiki.openstreetmap.org/wiki/OSM_tags_for_routing/Maxs
 
 ## Enabling Rules
 
-I you have a working GraphHopper setup it is easy to enable Spatial Rules, **but they only work with the DataFlagEncoder**.
-We provide a set of approximate country borders, within the GraphHopper repository. If you need exact borders you can
-get the exact borders from [here](https://github.com/datasets/geo-countries). Go to your
-`config.yml` file and
-uncommend the line: `spatial_rules.borders_directory` and point it to the directory where your rules are. You need to re-import your graph after that.
+I you have a working GraphHopper setup it is easy to enable Spatial Rules, **but they only work with the
+DataFlagEncoder**. We provide a set of approximate country borders, within the GraphHopper repository. If you need exact
```

Comment:
We can remove the bold phrase as it should work regardless of the FlagEncoder.

Reply:
-> https://github.com/graphhopper/graphhopper/pull/2367